docs/book/getting-started/installation/installation.md

```diff
@@ -58,15 +58,6 @@ print(zenml.__version__)
 If you would like to learn more about the current release, please visit our 
 [PyPi package page.](https://pypi.org/project/zenml)
 
-## Known installation issues for M1 Mac Users
```

Comment:
I would keep this here and add a link to the bigger doc page

Reply:
Makes sense, done.